### DISCUSSION TITLE
Should tests of proportions live in `ttest`?

### DISCUSSION THREAD

**Discussion by cwolock** [statdivlab/rigr#44] [Ideas]
Currently proportion tests are part of `ttest`. On one hand, I understand that proportion tests are also tests of equality of means, so they are related to t-tests in that way (and I believe they are taught at roughly the same time as t-tests, at least in BIOST 511). However, it seems misleading to call them t-tests since they aren't based on the Student's t distribution. There is also an `exact` option for the proportion test, which doesn't really fit in with t-tests in my opinion. I would be in favor of having proportion tests live in their own function, with both exact and approximate options.

**Comment by taylorokonek** [comment]
I'm also in favor of having proportion tests live in their own function, for essentially the same reasons as you've mentioned.

**Reply by bdwilliamson** [reply]
I agree!
